### DISCUSSION TITLE
there is an xkcd comic having title text about imagemagick

### DISCUSSION THREAD

**Discussion by therealadityashankar** [ImageMagick/ImageMagick#2435] [General]
![xkcd comic](https://imgs.xkcd.com/comics/dependency.png)

title text: **Someday ImageMagick will finally break for good and we'll have a long period of scrambling as we try to reassemble civilization from the rubble.**

permalink: https://xkcd.com/2347/

**Comment by therealadityashankar** [comment]
I think this is his way of thanking you

**Reply by fmw42** [reply]
What does this have to do with ImageMagick. I think it is SPAM

**Reply by dlemstra** [reply]
It's not spam Fred. The title text has a reference to ImageMagick and looks like a thank you to us. Thank for the link @therealadityashankar

**Comment by fmw42** [comment]
ImageMagick is not maintained by someone in Nebraska. So that is what was especially bothering.

**Reply by ghost** [reply]
Well on the plus side it brought me to this wonderful app!  I had never heard of ImageMagick....
